subiquity/common/filesystem/gaps.py

```diff
@@ -41,6 +41,26 @@ class Gap:
     def id(self):
         return 'gap-' + self.device.id
 
+    def copy(self):
```

Comment:
A method like this clearly suggests that Gaps should be immutable. Do we want to consider using the attrs support for this? https://www.attrs.org/en/stable/examples.html#immutability

Reply:
Yes, that is reasonable, and with that `copy()` is not as helpful, so I ended up dropping `copy()`.